modules/items/client/controllers/list-items.client.controller.js

```diff
@@ -5,11 +5,38 @@
     .module('items')
     .controller('ItemsListController', ItemsListController);
 
-  ItemsListController.$inject = ['ItemsService'];
+  ItemsListController.$inject = ['ItemsService', 'CategoriesService', 'ModulesService'];
 
-  function ItemsListController(ItemsService) {
+  function ItemsListController(ItemsService, CategoriesService, ModulesService) {
     var vm = this;
 
     vm.items = ItemsService.query();
+    vm.cats = CategoriesService.query();
+    vm.mods = ModulesService.query();
+    vm.increment = function(item, inc){
+    	item.count+=inc;
+      var flag = false;
+      for(var i=0; i<vm.items.length; i++){
+        var comparator = vm.items[i];
+        if(item.title == comparator.title && item.content == comparator.content && item.created == comparator.created && item.user == comparator.user && item.restockThreshold == comparator.restockThreshold && item.workingStatus == comparator.workingStatus && item.categories == comparator.categories && item.modules == comparator.modules){
```

Comment:
so people can better read what you are doing, you probably want to use this function instead

https://docs.angularjs.org/api/ng/function/angular.equals